packages/discovery-provider/src/api/v1/models/playlists.py

```diff
@@ -124,7 +124,9 @@
     "playlist_full",
     full_playlist_without_tracks_model,
     {
-        "tracks": fields.List(fields.Nested(track_full), required=True),
+        # Fetching playlists by ids will give tracks, but fetching
```

Comment:
Shouldn't the endpoint for fetching playlists for a user use `FullPlaylistWithoutTracks` then? I think the two different types were meant to distinguish this case.

Reply:
yeah you're right. didn't read this code. thx. updating